### PR TITLE
New package: lms-3.29.1

### PR DESCRIPTION
**Pull request by Danoloan10**
LMS is a lightweight Subsonic compatible music server written in C++. It is developed using the Wt web framework.

#### General
- [x] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

#### Does it build and run successfully? 
- [x] I built this PR locally for my native architecture, (x86_64-glibc)
- [x] I built this PR locally for these architectures (if supported. mark crossbuilds):
  - [x] i686
  - [x] x86_64-musl
  - [x] aarch64
  - [x] aarch64-musl
  - [x] armv7l
  - [x] armv7l-musl
  - [x] armv6l
  - [x] armv6l-musl

### CODE REVIEW

**Comment by Danoloan10** [comment]
This package depends on #33222, and the commits related to Wt are those in that PR. Should I close it untill #33222 is merged, or keep it open this way?

**Comment by paper42** [comment]
> This package depends on #33222, and the commits related to Wt are those in that PR. Should I close it untill #33222 is merged, or keep it open this way?

You can close that PR and include the commit here.

**Comment by Chocimier** [comment]
- should check XBPS_TARGET_NO_ATOMIC8 for libatomic targets
- as seen in build log of checks, need to commit symlinks of subpackages
- changelog at https://webtoolkit.eu/wt/doc/reference/html/Releasenotes.html
- please squash shlibs into package commit, as well as above fixes

**Comment by Danoloan10** [comment]
will work on it asap, thank you

**Comment by Danoloan10** [comment]
Previous push was checked successfully by the workflow, but I forgot to add the changelog so I had to push again. I think all is fixed now.

**Comment by Danoloan10** [comment]
Rebased because there were too many commits ahead

**Comment by Danoloan10** [comment]
x86_64-musl build is failing python 3.10 is failing

**Comment by Danoloan10** [comment]
Is there anything wrong with this package?

**Comment by Chocimier** [comment]
It's fine, just forgot about it.

- files in /etc should be marked as `conf_files`
- wt got updated since
- don't set CMAKE_BUILD_TYPE, xbps-src passes its own optimization flags
- leave BUILD_TESTING on for native build

**Comment by Danoloan10** [comment]
LMS itself has also recently been updated. Also a PR was merged to this new version with tests disabled when cross-compiling.

**Comment by Danoloan10** [comment]
Wt was updated again

**Comment by Danoloan10** [comment]
I do not understand why the checks have failed, I can't see the error in the logs.

**Comment by paper42** [comment]
> Your branches is based on too old copy.
> Please rebase to newest copy.
> Error: Process completed with exit code 1.

rebase on top of master

**Comment by Danoloan10** [comment]
thanks will do

**Comment by Chocimier** [comment]
> I can't see the error in the logs.

You should be able to look at logs, just click on red cross indicator or "Details" of particular job.

**Comment by Danoloan10** [comment]
Wt was updated again: 4.7.0

File: srcpkgs/lms/template [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Template file for 'lms'
+pkgname=lms
+version=3.28.0
+revision=1
+build_style=cmake
+hostmakedepends="pkg-config"
+makedepends="wt-devel ffmpeg-devel boost-devel libconfig++-devel taglib-devel pam-devel libgraphicsmagick-devel gtest-devel"
+conf_files="/etc/lms.conf /etc/pam/lms"
```
Comment (Chocimier) [inline srcpkgs/lms/template:8]: Void uses `/etc/pam.d`

**Review by Chocimier — COMMENTED** [review]
Doesn't it need an runit service?

**Comment by Danoloan10** [comment]
The runit service is a good idea, I'd swear i did it already but I didnt. Fixing that and the pam.d asap.

11 mar 2022 22:19 por ***@***.***:

**Comment by Chocimier** [comment]
Great! Consider using `system_accounts=_lms` and `make_dirs=/var/lms` for the service.

**Comment by Danoloan10** [comment]
First attempt at making the service done

**Comment by Danoloan10** [comment]
Wt was updated again

**Comment by Danoloan10** [comment]
Wt was updated again

**Pull request closed without merging**